src/patternfly/components/Dropdown/dropdown.hbs

```diff
@@ -7,23 +7,7 @@
   {{else if dropdown--IsMultiSelect}}
     <span id="{{id}}-label" hidden>Select one or more fruits</span>
   {{/if}}
-  <button id="{{id}}-button"
-    class="pf-c-dropdown__toggle{{#if dropdown-toggle--modifier}} {{dropdown-toggle--modifier}}{{/if}}"
-    {{#if dropdown--IsSelect}}
-      aria-labelledby="{{id}}-label {{id}}-button"
-    {{else if dropdown--IsMultiSelect}}
-        aria-labelledby="{{id}}-label {{id}}-button"
-    {{/if}}
-    aria-expanded="{{#if dropdown--IsExpanded}}true{{else}}false{{/if}}"
-    {{#if aria-label}} aria-label="{{aria-label}}" {{/if}}>
-    {{> @partial-block}}
-    {{#if dropdown--HasToggleIcon}}
-    {{> dropdown-toggle-icon}}
-    {{/if}}
-    {{#if dropdown--HasKebabIcon}}
-    {{> kebab-icon}}
-    {{/if}}
-  </button>
+  {{> dropdown-toggle}}
```

Comment:
Should there always be `{{> @partial-block}}`?

Reply:
This is the only one without a toggle. I'm maybe not doing it right,  but it doesn't seem needed because of the conditionals that contain the `{{> @partial-block}}`

Reply:
After talking, it seems like we don't need to enforce that there's always {{> @partial-block}} so this is good as is.